pkgs/tools/system/runit/default.nix

```diff
@@ -34,8 +38,10 @@ stdenv.mkDerivation rec {
     cd src
 
     # Both of these are originally hard-coded to gcc
-    echo cc > conf-cc
-    echo cc > conf-ld
+    echo ${stdenv.cc.targetPrefix}cc > conf-cc
+    echo ${stdenv.cc.targetPrefix}cc > conf-ld
+    export AR=${stdenv.cc.targetPrefix}ar
```

Comment:
Hmm, I suppose this also isn't needed anymore.

Reply:
Oh good point